### PR TITLE
RK-12482 - Change yarn script build name

### PR DESCRIPTION
**Pull request by Gilwe**
Self explanatory, this isn't publishing

### CODE REVIEW

**Review by adirook — DISMISSED** [review]

File: .circleci/config.yml [hunk 1]
```diff
@@ -50,7 +50,7 @@ jobs:
             echo $WIN_EV_CERT_BASE64 | base64 --decode > rookout.crt
             export WINDOWS_EV_CERTIFICATE_PATH=rookout.crt
             # package code for every distribution (mac, win, linux) and publish as github release
-            yarn run mac-publish
+            yarn run build-packages
```
Comment (Urook) [inline .circleci/config.yml:53]: Can you please add a descrption for this PR?

**Review by Urook — CHANGES_REQUESTED** [review]

File: package.json [hunk 2]
```diff
@@ -11,7 +11,7 @@
     "package-windows": "electron-builder --win",
     "build-headless": "tsc",
     "dist": "electron-builder",
-    "mac-publish": "export ELECTRON_BUILDER_ALLOW_UNRESOLVED_DEPENDENCIES=true && electron-builder --mac --win --linux -p always",
+    "build-packages-all-distributions": "export ELECTRON_BUILDER_ALLOW_UNRESOLVED_DEPENDENCIES=true && electron-builder --mac --win --linux -p always",
```
Comment (Urook) [inline package.json:14]: Please change version to 1.8.39. I will publish a new minor shortly, so this may remain as a draft but still be helpful.

**Review by Urook — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (Gilwe) [reply package.json:14]: what's the reason for increasing the version? there's zero impact on behavior, even on the ci.

**Review by Urook — APPROVED** [review]

**Pull request merged**